modules/service/src/main/resources/db/migration/V0030__observations.sql

```diff
@@ -14,19 +88,45 @@ create type e_obs_status as enum(
 
 create type e_obs_active_status as enum('active', 'inactive');
 
-CREATE DOMAIN d_observation_id AS VARCHAR; -- TODO format check
+create domain d_observation_id as varchar; -- TODO format check
```

Comment:
I lower cased these to match the others.  Since I updated instead of doing a proper migration, you'll need to obliterate your database and build from scratch.